### PR TITLE
[RAC-6404] fix Dockerfile

### PR DESCRIPTION
**Pull request by mcgG**
**Background**
----
Dockerfile update is needed due to new ucs-service code
[https://rackhd.atlassian.net/browse/RAC-6404](https://rackhd.atlassian.net/browse/RAC-6404)

**Detail**
----
New Dockerfile created.
Using supervisord to start the ucs-service see docker-entrypoint.sh.
Changed supervisord.config to prevent supervisord from running as daemon

@pengz1 @bbcyyb @PengTian0

### CODE REVIEW

**Comment by RackHDCI** [comment]
The Copyright Check has failed!
The files listed below do not have a valid copyright statement:
- Dockerfile - docker-entrypoint.sh - requirements.txt - supervisord.conf

**Comment by RackHDCI** [comment]
The Copyright Check has failed!
The files listed below do not have a valid copyright statement:
- docker-entrypoint.sh - requirements.txt - supervisord.conf

File: supervisord.conf [hunk 1]
```diff
@@ -8,6 +8,7 @@ pidfile=/var/run/supervisord.pid ; supervisord pidfile; default supervisord.pid
 nodaemon=false               ; start in foreground if true; default false
 minfds=1024                  ; min. avail startup file descriptors; default 1024
 user=root
+nodaemon=true
```
Comment (pengz1) [inline supervisord.conf]: This will be annoying for user who use it outside docker.

**Comment by RackHDCI** [comment]
The Copyright Check has failed!
The files listed below do not have a valid copyright statement:
- docker-entrypoint.sh

File: README.md [hunk 2]
```diff
@@ -52,6 +52,11 @@ To run unit tests:
 
     python -m unittest discover -s tests
 
+## Running in docker
+Ucs-service in docker shouldn't run in host mode, because rabbitmq service is built in docker and it may conflict with RackHD's rabbitmq server.
+    sudo docker -t example/ucs-service .
```
Comment (pengz1) [inline README.md]: Add a blank line here.

File: docker-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Copyright 2017, Dell EMC, Inc.
+
+service rabbitmq-server start
+python app.py >> .app.log &
+python tasks.py worker >> tasks.log
```
Comment (pengz1) [inline docker-entrypoint.sh]: Align your log format.

**Pull request merged**